### PR TITLE
Modify rule S6330: Default Queue encryption is now SSE-SQS (APPSEC-339)

### PR DESCRIPTION
**Pull request by sebastien-andrivet-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [x] logical errors and incorrect information
- [x] information gaps and missing content
- [x] text style and tone
- [x] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

**Comment by hendrik-buchwald-sonarsource** [comment]
Hey, I got an email about this because I was watching the ticket (since I created it). I didn't look at the actual changes but I noticed that the metadata of the PR needs to be slightly adjusted. Mainly, the title of the PR and it needs a label. Thanks!

You can find the details in the readme: https://github.com/SonarSource/rspec#to-modify-an-existing-rule

File: rules/S6330/terraform/rule.adoc [hunk 1]
```diff
@@ -22,8 +23,8 @@ For https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/
 [source,terraform]
 ----
 resource "aws_sqs_queue" "queue" {
-  name = "sqs-encrypted"
-  kms_master_key_id = aws_kms_key.enc_key.key_id
+  name = "sqs-unencrypted"
+  sqs_managed_sse_enabled = true # Sensitive, encryption disabled
```
Comment (egon-okerman-sonarsource) [inline rules/S6330/terraform/rule.adoc:27]: I think there is a typo here

**Review by egon-okerman-sonarsource — CHANGES_REQUESTED** [review]
One small error should be fixed, the rest looks good.

File: rules/S6330/terraform/rule.adoc (hunk 1 above)
Comment (sebastien-andrivet-sonarsource) [reply rules/S6330/terraform/rule.adoc]: Yes, you're right.

**Review by egon-okerman-sonarsource — APPROVED** [review]
LGTM, let's hold off on merging for now until this fix gets implemented.

**Comment by petertrr** [comment]
I removed the `iac` label as it has been merged separately

**Comment by egon-okerman-sonarsource** [comment]
This has been implemented in both IAC and Python for a while now, merging it.

**Pull request merged**